core/trino-main/src/main/java/io/trino/sql/ParameterUtils.java

```diff
@@ -32,7 +32,7 @@
 {
     private ParameterUtils() {}
 
-    public static Map<NodeRef<Parameter>, Expression> parameterExtractor(Statement statement, List<Expression> parameters)
+    public static Map<NodeRef<Parameter>, Expression> bindParameters(Statement statement, List<Expression> parameters)
```

Comment:
Can we be a bit more specific here like `bindParametersToExpression` the method name might mislead us `ParameterRewriter` - where we bind the `parameter` to `expression`

Reply:
I don't think the "toExpression" suffix adds much value. Although, it'd be clearer if the second argument were named "values" instead of parameters -- the parameters are in the query itself.